warmup.krun

```diff
@@ -140,21 +140,27 @@ SKIP = [
 N_EXECUTIONS = 10  # Number of fresh processes.
 
 if sys.platform.startswith("linux"):
+    # Assumes systemd is used for init.
     PRE_EXECUTION_CMDS = [
-        "sudo service cron stop",
-        "sudo service atd stop",
-        "sudo service postfix stop",
-        "sudo service networking stop",
+        # Stop network first. If an interface accepts a DHCP lease during one
+        # of the later commands below, it can cause the command to be
+        # "cancelled" by systemd. Bringing the network itself down can fail in
+        # the same way, so try up to 5 times (with sleeps between) before
+        # finally giving up.
+        "sudo sh -c 'for i in 1 2 3 4 5; do systemctl stop networking && break; sleep 10; false; done'",
```

Comment:
Why 5? Why not infinite? Also, having a ping means that we can quit as soon as its worked.